source/analysis/analysis.do

```diff
@@ -2,17 +2,23 @@ version 14
 clear all
 set more off
 
+
 program main
-    import delimited "build/data/data.txt", delimiters("|") varnames(1) clear
+    yaml read YAML using constants.yaml
```

Comment:
Why do we prefer to read the yaml in stata within the main function but before the main function in R?